api/institutions/serializers.py

```diff
@@ -41,6 +41,7 @@ class InstitutionSerializer(JSONAPISerializer):
         ser.CharField(read_only=True),
         permission='view_institutional_metrics',
     )
+    can_request_access = ser.CharField(read_only=True)
```

Comment:
Make it clear this refers to a specific feature. Refactor to something like `institutional_request_access_enabled`